### PR TITLE
Fail hard if configured master file is not present

### PR DESCRIPTION
**Pull request by BjRo**
@boosty Any objects?

### CODE REVIEW

File: test/beetle/deduplication_store_test.rb [hunk 1]
```diff
@@ -68,7 +68,7 @@ def teardown
 
     test "should return nil if the master file doesn't exist" do
       Beetle.config.redis_server = "/tmp/__i_don_not_exist__.txt"
-      assert_equal nil, @store.redis_master_from_master_file
+      assert_raises(Errno::ENOENT) { @store.redis_master_from_master_file }
```
Comment (boosty) [inline test/beetle/deduplication_store_test.rb]: You should update the test description accordingly :wink:

**Comment by boosty** [comment]
I like it! Actually I cannot think of a good reason not to raise the exception if the file is configured.

**Comment by BjRo** [comment]
Good, I'm going to merge and publish a new version

**Comment by BjRo** [comment]
Merged via https://github.com/xing/beetle/commit/db3460f9bf93f5d033c5012de5a0cc9ef7a91d05

**Pull request closed without merging**